R/correctionsataglance-render.R

```diff
@@ -41,7 +41,8 @@ correctionsataglanceReport <- function(reportObject) {
   dataLanes <- allLaneData[['dataLanes']]
   processOrderLabelYPos <- mean(c(dataLanes[['preData']][["laneNameYPos"]],
                                   dataLanes[['normalData']][["laneNameYPos"]],
-                                  dataLanes[['postData']][["laneNameYPos"]]))
+                                  dataLanes[['postData']][["laneNameYPos"]],
+                                  dataLanes[['thresholdData']][["laneNameYPos"]]))
```

Comment:
This part of the code is adjusting the y position of the "processing order" label to be in the middle of the processing order lanes. I'm not sure if threshold data is supposed to go under the "Processing Order" label or not? I don't think it was under it before when the lane show up, though I'm not totally sure.